alarm/management/commands/price_break_threshold_alarm.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+
+from alarm.models import Coin
+from alarm.utils import connect_binance_socket, process_binance_messages, close_binance_sockets
+
+
+class Command(BaseCommand):
+    help = 'Starts streaming Binance market data'
+
+    def handle(self, *args, **options):
+        # Define the list of currencies and intervals you want to stream
+        currencies = Coin.objects.values_list('coin_abbreviation', flat=True)
+        intervals = Coin.objects.values_list('interval', flat=True)
```

Comment:
This seemed like a good idea to me, because we need an interval to each socket. And if we need 2 coins to search or, for example, 3 coins for each coin we need an interval or there will be no search